azurerm/resource_arm_automation_account.go

```diff
@@ -169,8 +188,21 @@ func resourceArmAutomationAccountRead(d *schema.ResourceData, meta interface{})
 		d.Set("location", azureRMNormalizeLocation(*location))
 	}
 
-	if err := d.Set("sku", flattenAutomationAccountSku(resp.Sku)); err != nil {
-		return fmt.Errorf("Error setting `sku`: %+v", err)
+	if skuName == "" {
```

Comment:
What you can do is mark both as computed until 2.0 and just set both instead of doing tricky logic like this

Reply:
Would that cause issues when we totally remove `sku`?